### PR TITLE
Prepare Playground plugin for publishing

### PR DESCRIPTION
**Pull request by bgrgicak**
## What?

Updates the readme to match WordPress.org expectations and adds content export filters to allow skipping of sensitive or unused files. 

## Why?

We want to publish this plugin to WordPress.org.

## How?

By updating the readme and adding filters to the data exporter.

## Testing Instructions

1. Check out the branch.
2. Add the Playground plugin to a WordPress site
3. Follow the Usage instructions in readme.txt

### CODE REVIEW

File: packages/playground/README.txt [hunk 1]
```diff
@@ -0,0 +1,57 @@
+=== WordPress Playground ===
+Contributors: antoniosejas, berislavgrgicak, zieladam
```
Comment (bgrgicak) [inline packages/playground/README.txt]: @seanmorris what's your WordPress.org username? I would like to include you as a contributor.

File: packages/playground/src/playground-zip.php [hunk 2]
```diff
@@ -32,6 +32,9 @@ function zip_wp_content($zip)
 			continue;
 		}
 
+		if (apply_filters('playground_zip_exclude_file', false, $file)) {
```
Comment (adamziel) [inline packages/playground/src/playground-zip.php]: Or

```suggestion
		$file = apply_filters('playground_exported_file', $file);
		if ( false !== $file ) {
```

This way a plugin could modify the file contents if it wanted to

File: packages/playground/README.txt (hunk 1 above)
Comment (adamziel) [reply packages/playground/README.txt]: Let's start that list with `wordpressdotorg,` similarly to [what the classic-editor plugin does](http://plugins.svn.wordpress.org/classic-editor/trunk/readme.txt).

Comment (adamziel) [reply packages/playground/README.txt]: Also let's format this entire file using the wiki syntax with `==` for headers, also similarly to that file referenced above. I'll do the same for block plugin.

File: packages/playground/README.txt [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+Tested up to: 6.4
+Stable tag: 0.1.0
+Requires PHP: 7.0
+License: GPLv2
+License URI: https://www.gnu.org/licenses/gpl-2.0.html
+Early preview: Enables running a sandbox of your site using WordPress Playground (https://github.com/WordPress/wordpress-playground)
+
+# WordPress Playground
+
+_v0.1.0_
```
Comment (adamziel) [inline packages/playground/README.txt]: Let's label this as Changelog and format similarly to the Classic Editor readme http://plugins.svn.wordpress.org/classic-editor/trunk/readme.txt

File: packages/playground/README.txt (hunk 1 above)
Comment (bgrgicak) [reply packages/playground/README.txt]: I updated both readme files

File: packages/wordpress-playground-block/README.plugindirectory.txt [hunk 4]
```diff
@@ -1,5 +1,5 @@
 === WordPress Playground Block ===
-Contributors: dawidurbanski, zieladam
+Contributors: wordpressdotorg, dawidurbanski, zieladam
```
Comment (adamziel) [inline packages/wordpress-playground-block/README.plugindirectory.txt:2]: Thanks!

**Comment by adamziel** [comment]
Good work, thank you!

**Pull request merged**